stuffer/s2n_stuffer_text.c

```diff
@@ -49,6 +49,22 @@ int s2n_stuffer_skip_whitespace(struct s2n_stuffer *s2n_stuffer)
     return skipped;
 }
 
+/* Skips the stuffer until the first instance of the target character or until there is no more data. */
+int s2n_stuffer_skip_to_char(struct s2n_stuffer *stuffer, char target)
+{
+    while (s2n_stuffer_data_available(stuffer) > 0) {
+        char c;
+        GUARD(s2n_stuffer_peek_char(stuffer, &c));
+        if (c == target) {
+            break;
+        }
+
+        stuffer->read_cursor++;
```

Comment:
It might be more future proof to use s2n_stuffer_read_uint8 here; in case we add side-effects like reading from a file and so on.